catalogs/catalog-jdbc-doris/build.gradle.kts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+description = "catalog-jdbc-doris"
+
+plugins {
+  `maven-publish`
+  id("java")
+  id("idea")
+}
+
+dependencies {
+  implementation(project(":api"))
+  implementation(project(":catalogs:catalog-jdbc-common"))
+  implementation(project(":common"))
+  implementation(project(":core"))
+
+  implementation(libs.bundles.log4j)
+  implementation(libs.commons.collections4)
+  implementation(libs.commons.lang3)
+  implementation(libs.guava)
+  implementation(libs.jsqlparser)
+  implementation(libs.slf4j.api)
+
+  testImplementation(project(":catalogs:catalog-jdbc-common", "testArtifacts"))
+  testImplementation(project(":clients:client-java"))
+  testImplementation(project(":integration-test-common", "testArtifacts"))
+  testImplementation(project(":server"))
+  testImplementation(project(":server-common"))
+
+  testImplementation(libs.commons.lang3)
+  testImplementation(libs.guava)
+  testImplementation(libs.junit.jupiter.api)
+  testImplementation(libs.junit.jupiter.params)
+  testImplementation(libs.mysql.driver)
+  testImplementation(libs.testcontainers)
+  testImplementation(libs.testcontainers.mysql)
+
+  testRuntimeOnly(libs.junit.jupiter.engine)
```

Comment:
We can remove the dependencies listed above because they are not currently in use.

Reply:
removed

Reply:
If we need to remove it, I believe other dependencies like test container should also be dropped.

Reply:
@yuqi1129 yes，I had remove all test dependencies